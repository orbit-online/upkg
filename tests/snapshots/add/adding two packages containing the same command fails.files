[drwxr-xr-x]  .
├── [drwxr-xr-x]  .upkg
│   ├── [drwxr-xr-x]  .bin
│   │   └── [lrwxrwxrwx]  acme.bin -> ../.packages/acme-empty@db82dc5f57b1da1024134f1793d6cc34b7f0b76b45d910426fe292e3a623a51b/bin/acme.bin
│   ├── [drwxr-xr-x]  .packages
│   │   └── [drwxr-xr-x]  acme-empty@db82dc5f57b1da1024134f1793d6cc34b7f0b76b45d910426fe292e3a623a51b
│   │       ├── [drwxr-xr-x]  bin
│   │       │   └── [-rwxr-xr-x]  acme.bin
│   │       └── [-rw-r--r--]  upkg.json
│   └── [lrwxrwxrwx]  acme-empty -> .packages/acme-empty@db82dc5f57b1da1024134f1793d6cc34b7f0b76b45d910426fe292e3a623a51b
└── [-rw-r--r--]  upkg.json
