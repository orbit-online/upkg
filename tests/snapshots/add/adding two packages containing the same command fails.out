upkg: conflict: 'acme-empty-v1.0.2-no-metadata.tar' and 'acme-empty' both have a command named 'acme-empty-v1.0.2.bin'
upkg: An error occurred while installing '$BATS_RUN_TMPDIR/package-fixtures/acme-empty-v1.0.2-no-metadata.tar'
